pkg/allocation/converters/converter_test.go

```diff
@@ -355,6 +375,10 @@ func TestConvertAllocationResponseToGSA(t *testing.T) {
 				Ports: []*pb.AllocationResponse_GameServerStatusPort{},
 			},
 			want: &allocationv1.GameServerAllocation{
```

Comment:
Should there be a test where the typemeta isn't defined as part of `in` and we verify it gets added to `want`?

Reply:
`TestConvertAllocationResponseToGSA` is exactly doing it. The is an empty input and want has the `TypeMeta`.

Reply:
Sorry I missed that. I thought all of the additions were to input, but I see that this last one is for output.